grafana-plugin/.eslintrc.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
     {
       files: ['src/**/*.{ts,tsx}'],
       rules: {
-        'deprecation/deprecation': 'off',
+        'deprecation/deprecation': 'warn',
```

Comment:
isn't it warn by default? If so we can just remove  whole `rules` from here

Reply:
First time I attempt this by removing the rule completely didn't trigger any warnings/errors 🤔